packages/block-library/src/query-title/block.json

```diff
@@ -29,6 +29,11 @@
 			"default": true
 		}
 	},
+	"example": {
+		"attributes": {
+			"type": "search"
+		}
+	},
```

Comment:
I believe the block variations (for Archive and Search Results titles) override the type attribute so they don't need their own custom example definitions within `variations.js`